components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -26,38 +26,41 @@ import {
 import { LayoutFacadeService, Sidebar } from 'app/entities/layout/layout.facade';
 import { saveAs } from 'file-saver';
 import {
-  Chicklet
+  Chicklet,
+  ReportingFilterTypes
 } from '../../../types/types';
 import { DateTime } from 'app/helpers/datetime/datetime';
 import { pickBy } from 'lodash/fp';
 import { FilterC } from './types';
 
+
 @Component({
   templateUrl: './reporting.component.html',
   styleUrls: ['./reporting.component.scss'],
   providers: [SuggestionsService]
 })
+
 export class ReportingComponent implements OnInit, OnDestroy {
```

Comment:
I wanted stronger typing in for my `ReturnParams` interface, which required me to reach out a little further and add some stronger types to the filters overall.